tests/acceptance-bdd/literature_suggestion.feature

```diff
@@ -0,0 +1,287 @@
+    And I insert Mister Brown in the input box with id authors-1-name
+    And I insert Lorem ipsum dolor sit amet, consetetur sadipscing elitr. in the input box with id abstract
+    And I click on the submit button
+    And I go to the holding panel list page
+    And I should see the record in the list page
+    And I click on the button link with text Show abstract
+    Then I should see Computing in the record entry
+    And I should see My Title For Test in the record entry
+    And I should see admin@inspirehep.net in the record entry
+    And I should see Mister White; Mister Brown in the record entry
+    And I should see Lorem ipsum dolor sit amet, consetetur sadipscing elitr. in the record entry
+
+  Scenario: Submit thesis and verify record detail in the detail page
+    Given I am logged in
+    When I go to the literature suggestion page
+    And I click on the button Skip, and uncollapse all panels
+    And I select the value thesis in the select box with id type_of_doc
+    And I insert My Title For Test in the input box with id title
+    And I select the value Computing in the subject dropdown list
+    And I insert Mister White in the input box with id authors-0-name
+    And I insert Wisconsin U., Madison in the input box with id authors-0-affiliation
+    And I insert Lorem ipsum dolor sit amet, consetetur sadipscing elitr. in the input box with id abstract
+    And I click on the submit button
+    And I go to the holding panel list page
+    And I should see the record in the list page
+    And I click on the button link with text My Title For Test
+    Then I should see Lorem ipsum dolor sit amet, consetetur sadipscing elitr. in the record detail
+    And I should see Submitted by admin@inspirehep.net in the record detail
+    And I should see Wisconsin U., Madison in the record detail
+    And I should see My Title For Test in the record detail
+    And I should see Mister White in the record detail
+    And I should see Computing in the record detail
+
+
+  Scenario: Accept thesis and verify the confirmation message
+    Given I am logged in
+    When I go to the literature suggestion page
+    And I click on the button Skip, and uncollapse all panels
+    And I select the value thesis in the select box with id type_of_doc
+    And I insert My Title For Test in the input box with id title
+    And I select the value Computing in the subject dropdown list
+    And I insert Mister White in the input box with id authors-0-name
+    And I insert Lorem ipsum dolor sit amet, consetetur sadipscing elitr. in the input box with id abstract
+    And I click on the submit button
+    And I go to the holding panel list page
+    And I should see the record in the list page
+    And I click on the button link with text My Title For Test
+    And I click on the accept button
+    Then I should see the message of confirmation
+
+
+  Scenario: Submit article and verify record detail in the list page
```

Comment:
Above you call `record detail` the detailed page of the Holding Pen, might be confusing to call it the same the entry in the search results.